app/assets/sass/components/cookie-message.scss

```diff
@@ -1,19 +1,112 @@
-.cookie-banner {
-  box-sizing: border-box;
```

Comment:
Will the old banner work for now as well?

Reply:
we don't use this class any more